src/Microsoft.IdentityModel.JsonWebTokens/JsonWebToken.cs

```diff
@@ -116,7 +116,7 @@ public IEnumerable<string> Audiences
                         return value.ToObject<List<string>>();
```

Comment:
In order to address issue #981, we also need to check if 'value' is a null reference. The JObject.GetValue(string) method returns null if the property name does not exist.

Reply:
One approach would be to use the IS keyword here. For example, the 'if (Payload != null)' could be replaced by 'if (Payload?.GetValue(JwtRegisteredClaimNames.Aud) is JToken value)'

Reply:
Thanks for pointing this out. :) I just made the change.